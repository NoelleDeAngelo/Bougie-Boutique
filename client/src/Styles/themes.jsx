import styled, { createGlobalStyle } from 'styled-components';

export const Body = createGlobalStyle`
  body {
    color: ${props => props.theme.text};
    background: ${props => props.theme.bg};
    font-family: 'Josefin Sans', sans-serif;
    font-weight: 500;
    font-size: 18px;
    margin: 0px;
  }

  #content {
    padding: 0px 20px;
  }

  a {
    color: #0056c5;
    transition: color 0.5s;
  }
  a: hover {
    color: ${props => props.theme.text};
  }
  input {
    -webkit-appearance: none;
    appearance: none;
  }
`;

export const ThemeToggle = styled.button`
  border: white;
  color: #ccc;
  background: none;
  font-size: 20px;
  outline: none;
`;

export const Title = styled.h1`
  font-family: 'Yeseva One', cursive;
  margin: 0px;
`;

export const light = {
  border: 'white',
  outline: '#002a60',
  text: 'black',
  bg: 'white',
};

export const dark = {
  border: '#002a60',
  outline: '#cccccc',
  text: '#cccccc',
  bg: '#111111',
}

