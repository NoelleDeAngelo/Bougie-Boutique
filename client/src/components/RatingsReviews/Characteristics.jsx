import React from 'react';



var CharacteristicDeff = {
  Size: ['A size too small', '1/2 size too small', 'Perfect', '1/2 size too big,', 'a size too wide'],
  Width: ['Too narow', 'Slightly narrow', 'Perfect', 'Slightly wide', 'Too Wide'],
  Comfort: ['Uncomfortable', 'Slightly uncomfortable', 'Ok', 'Comfortable', 'Perfect'],
  Quality: ['Poor', 'Below average', 'What I expected', 'Pretty great', 'Perfect'],
  Length: ['Runs short', 'Runs slightly short', 'Perfect', 'Runs slightly long', 'Runs long'],
  Fit: ['Runs tight', 'Runs slightly tight', 'Perfect', 'Runs slightly long', 'Runs long']

};


const Characteristics = ({characteristics}) => {
return (<div>
    {Object.keys(characteristics).map((characteristic)=> {
      var rating = (characteristics[characteristic].value *50) - 50
     return ( <div style={{marginBottom:'50px', width:'250px', borderRadius: '4px'}}>
       <div>{characteristic}</div>
        <i style={{marginLeft:`${rating}px` }}className="lni lni-diamond"></i>
        <div style={{background:'#EEEEEE', width:'250px', height: '7px' }}></div>
        <span style={{fontSize:'90%', marginTop:'5px'}}>{CharacteristicDeff[characteristic][0]}</span>
        <span style={{float: 'right', fontSize:'90%', marginTop:'5px' }}>{CharacteristicDeff[characteristic][4]}</span>
      </div>)
  })}
  </div>)
}






export default Characteristics;
