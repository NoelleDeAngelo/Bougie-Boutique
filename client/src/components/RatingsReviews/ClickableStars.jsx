import React from 'react';

const ClickableStars = ({ starStyles, highlightStars }) => (
<span style={{'fontFamily': 'FontAwesome', 'color':'#DFDFDF' }}id= 'starRating'>
        <label ><i style ={starStyles.one} class="far fa-star"></i><input style ={{'display': 'none'}} type="radio" name="overallRating" value="1" id='1' onClick= {(event)=> highlightStars(event.target.value)} required></input></label>
        <label><i style ={starStyles.two} class="far fa-star"></i><input style ={{'display': 'none'}} type="radio" name="overallRating" value="2" id='2' onClick= {(event)=> highlightStars(event.target.value)}></input></label>
        <label><i  style ={starStyles.three} class="far fa-star"></i><input style ={{'display': 'none'}} type="radio" name="overallRating" value="3" id='3' onClick= {(event)=> highlightStars(event.target.value)}></input></label>
        <label><i  style ={starStyles.four} class="far fa-star"></i><input style ={{'display': 'none'}} type="radio" name="overallRating" value="4" id='4' onClick= {(event)=> highlightStars(event.target.value)}></input></label>
        <label><i  style ={starStyles.five} class="far fa-star"></i><input style ={{'display': 'none'}} type="radio" name="overallRating" value="5" id='5' onClick= {(event)=> highlightStars(event.target.value)}></input></label>
      </span>
);

export default ClickableStars;
