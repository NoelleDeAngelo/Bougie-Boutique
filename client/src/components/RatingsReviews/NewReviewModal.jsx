import React from 'react';
import Modal from 'react-modal';
Modal.setAppElement('#app');



var CharacteristicDeff = {
    Size: ['A size too small', '1/2 size too small', 'Perfect', '1/2 size too big,', 'a size too wide'],
    Width: ['Too narow', 'Slightly narrow', 'Perfect', 'Slightly wide', 'Too Wide'],
    Comfort: ['Uncomfortable', 'Slightly uncomfortable', 'Ok', 'Comfortable', 'Perfect'],
    Quality: ['Poor', 'Below average', 'What I expected', 'Pretty great', 'Perfect'],
    Length: ['Runs short', 'Runs slightly short', 'Perfect', 'Runs slightly long', 'Runs long'],
    Fit: ['Runs tight', 'Runs slightly tight', 'Perfect', 'Runs slightly long', 'Runs long']

  };

const AddCharRateing = (Characteristic, value) => {
  document.getElementById(Characteristic).innerHTML = CharacteristicDeff[Characteristic][value-1]
}

const generateCharacteristic = (Characteristic)=> {
  return (
    <div style={{'marginTop': '25px'}}><br></br>
      <span> * </span><u>{Characteristic}:</u> <span  style= {{'fontSize': '90%'}} id= {Characteristic}></span><br></br><br></br>
      <label style= {{'fontSize': '85%'}} >{CharacteristicDeff[Characteristic][0]} </label>
      <input onClick= {(event)=> AddCharRateing( Characteristic, event.target.value) } type="radio" name={Characteristic} value="1" ></input>
      <input onClick= {(event)=> AddCharRateing( Characteristic, event.target.value) } type="radio" name={Characteristic} value="2" ></input>
      <input onClick= {(event)=> AddCharRateing( Characteristic, event.target.value) } type="radio" name={Characteristic} value="3" ></input>
      <input onClick= {(event)=> AddCharRateing( Characteristic, event.target.value) } type="radio" name={Characteristic} value="4" ></input>
      <input onClick= {(event)=> AddCharRateing( Characteristic, event.target.value) } type="radio" name={Characteristic} value="5" ></input>
      <label style= {{'fontSize': '85%'}} > {CharacteristicDeff[Characteristic][4]}</label>
    </div>
  )
}

const NewReviewModal = ({close, isOpen, productName, characteristics})=> {
  return(
     <Modal isOpen= {isOpen} onRequestClose= {()=> close()} style={{
    'overlay': {'background':'grey'},
    'content': {'color':'black', 'width': '450px', 'margin':'auto'} }}>
    <span style={{'float': 'right', 'fontSize': '150%'}} onClick= {()=> close()}>&#10006;</span>
    <h2>Write Your Review</h2>
    <h3>About the {productName}</h3>
    <form >
      <label style={{'marginTop': '40px'}} for='starRating'>* Overall Rating:  </label>
      <span id= 'starRating'>
        <input type="radio" name="overallRating" value="1"></input>
        <input type="radio" name="overallRating" value="2"></input>
        <input type="radio" name="overallRating" value="3"></input>
        <input type="radio" name="overallRating" value="4"></input>
        <input type="radio" name="overallRating" value="5"></input>
      </span><br></br><br></br>
      <div style={{'marginTop': '25px'}}>  Would you recomended this product?</div>
      <input type="radio" name="recomended" value="Yes" id= 'yes'></input>
      <label for="yes">Yes</label><br></br>
      <input type="radio" name="recomended" value="No" id= 'no'></input>
      <label for="no">No</label>

      {Object.keys(characteristics).map((key)=> generateCharacteristic(key))}

    </form>
  </Modal>
  )
}

export default NewReviewModal;