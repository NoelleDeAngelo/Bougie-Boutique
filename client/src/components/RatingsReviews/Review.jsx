import React from 'react';
//import Stars from './Styles/Stars.js'

const Review = ({ review }) => (
  <div>
    <p>rating {review.rating}</p>
    <span>{review.date}</span>
    <h5>{review.summary}</h5>
    <p>{review.body}</p>
    {review.recommended ? <p>I recomend this product</p> : <div></div>}
    {review.responseToReview ? <p>{review.responseToReview}</p> : <div></div>}
    <span>Was this review helpful? <span>Yes({review.helpfulness})</span></span><span>Report</span>

  </div>
);

export default Review;