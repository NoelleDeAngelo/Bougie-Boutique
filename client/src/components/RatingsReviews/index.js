import React from 'react'
import WidgetContainer from '../../Styles'

const Reviews = ({store}) => (
<<<<<<< HEAD
  <div>I am a Reviews thing {store.state.currentProductId}</div>
=======
  <div>I am a Reviews thing {store.currentProductId}</div>
>>>>>>> rendering 2 questions on the QA widget
)

export default Reviews;