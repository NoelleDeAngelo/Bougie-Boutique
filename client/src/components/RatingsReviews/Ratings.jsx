import React from 'react';
import StarRating from './StarRatings';
import Stars from '../OverView/Styles/Stars.js';
import Characteristics from './Characteristics';

var calculatePercent = (obj)=>{
  var pos =Number(obj.true);
  var neg = Number(obj.false);
  var total = pos + neg;
  var percent = (pos / total) * 100;
  percent= Math.round(percent);
  return percent;

}


const Ratings = ({ meta }) => {
  if (!meta) {
     return <div>loading...</div>
   }
  return (<div>
    <div>{meta.averageRating}</div>
    <Stars rating={meta.starRating} />
    <p>{calculatePercent(meta.recommended)}% of reviews recomend this product</p>
    <Characteristics characteristics={meta.characteristics} />
  </div>);
}



export default Ratings;



// //recommended": {
//   "false": "3",
//   "true": "20"
// },