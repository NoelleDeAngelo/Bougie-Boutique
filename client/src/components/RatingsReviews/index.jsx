import React, {useState} from 'react';
// import WidgetContainer from '../../Styles'
import Ratings from './Ratings';
import Reviews from './Reviews';
import { RatingsContainer,  RatingsAndReviewsContainer} from '../../Styles'





class RatingsReviews extends React.Component {
  constructor(props){
    super(props);
    this.sortReviews= this.sortReviews.bind(this);
    this.filterReviews= this.filterReviews.bind(this);
    this.searchReviews= this.searchReviews.bind(this);
    this.state={
      currentProductId: null,
      reviewsToShow: this.props.store.state.reviews,
      sortedBy:0,
      filteredFor:{
        count:0,
        1:false,
        2:false,
        3:false,
        4:false,
        5:false
      }
    }
  }

  componentDidMount(){
    console.log('>------------------------ call to get reviews')
    this.props.store.setMeta(this.props.store.state.currentProductId)
    this.props.store.setReviews(this.props.store.state.currentProductId)
    this.setState({currentProductId: this.props.store.state.currentProductId})
  }

 componentDidUpdate(prevProps){
    if (Number(this.props.store.state.currentProductId) !== Number(this.state.currentProductId)){
      console.log('>------------------------ call to update reviews')
      this.setState({currentProductId: this.props.store.state.currentProductId})
      this.props.store.setMeta(this.props.store.state.currentProductId)
      this.props.store.setReviews(this.props.store.state.currentProductId)
    }
  }

  filterReviews(numberOfStars){
    var changeTo;
    var filteredFor = {...this.state.filteredFor};

    this.state.filteredFor[numberOfStars]? changeTo=false : changeTo=true;
    changeTo? filteredFor.count= filteredFor.count+1 : filteredFor.count= filteredFor.count-1;
    filteredFor[numberOfStars]=changeTo;
    filteredFor.count !== 0 ?  filteredFor= filteredFor : filteredFor= {
        count:0,
        1:false,
        2:false,
        3:false,
        4:false,
        5:false,
      }
    this.setState({filteredFor:filteredFor})
    if(filteredFor.count === 0){
      this.sortReviews(this.state.sortedBy,  this.props.store.state.reviews)
    }else {
      var reviewsToShow= []
    for (var i=0; i<this.props.store.state.reviews.length; i++){
      if (filteredFor[this.props.store.state.reviews[i].rating]){
        reviewsToShow.push(this.props.store.state.reviews[i]);
      }
    }
    this.sortReviews(this.state.sortedBy, reviewsToShow)
    }
  }

  sortReviews(index, inputArray){
    var array = inputArray || [...this.state.reviewsToShow];
    if(index===0){
      var storage={};
      array.sort((a,b)=> (a.date<b.date) ? 1: -1)
      for (var i=0; i<array.length; i++){
        storage[array[i].reviewerName]= i;
      }
      array= [...array]

      for (var i=0; i<array.length; i++){
        storage[array[i].reviewerName]= storage[array[i].reviewerName]- (array[i].helpfulness/4)
      }
      array.sort((a,b)=> (storage[a.reviewerName]> storage[b.reviewerName]) ? 1: -1)
      this.setState({sortedBy:0, reviewsToShow: array})
    }else if (index === 2){
      array.sort((a,b)=> (a.date<b.date) ? 1: -1)
      this.setState({sortedBy:2, reviewsToShow: array})
    }else if(index ===1){
      array.sort((a,b)=> (a.helpfulness<b.helpfulness) ? 1: -1)
      this.setState({sortedBy:1, reviewsToShow: array})
    }

  }

  searchReviews(term){
    if(term.length > 2){
      var searchedReviews=[];
      for (var i=0; i< this.state.reviewsToShow.length; i++){
        var currentReview = this.state.reviewsToShow[i]
        if (currentReview.body.indexOf(term) !== -1 || currentReview.summary.indexOf(term) !== -1 ){
          searchedReviews.push(currentReview);
        }
      }
      this.setState({reviewsToShow: searchedReviews})
    }else{
      this.setState({reviewsToShow: this.props.store.state.reviews})
    }
  }

  render(){
    if (this.state.reviewsToShow === undefined){
      return<div></div>
    }
    return (
      <section data-testid='content' tracking={'Ratings and Reviews'}>
      <h2 id="ratingsreviews" style={{marginBottom: '50px',marginTop: '100px', marginLeft:'3%'}}>Ratings and Reviews</h2>
      <RatingsAndReviewsContainer>
        <RatingsContainer>
          <Ratings
          meta={this.props.store.state.meta}
          filterReviews= {this.filterReviews}
          filteredFor= {this.state.filteredFor}/>
        </RatingsContainer>
        <div >
          <div style= {{ minWidth:'300px', height: '30px', width:'40%',border:`2px solid ${this.props.theme.blkGry}`, margin: '20px', marginLeft:'0px'}}> <i style= {{fontSize: '95%', margin: '3px'}} className="lni lni-32 lni-search-alt"></i><input placeholder= 'Search Reviews' style= {{height: '27px', width:'90%', border:`none`, outline:'none' , background:`${this.props.theme.invertWht}`, color: `${this.props.theme.blkGry}`}} type= 'text' onChange = {(event)=> this.searchReviews(event.target.value)}></input></div>
          <Reviews key={this.state.reviewsToShow}
            reviewsToShow ={this.state.reviewsToShow}
            product = {this.props.store.state.product}
            productId = {this.props.store.state.currentProductId}
            meta= {this.props.store.state.meta}
            setMeta= {this.props.store.setMeta}
            setReviews={this.props.store.setReviews}
            sortReviews= {this.sortReviews}
            theme={this.props.theme}
            searchReviews= {this.searchReviews}
            searchTerm={this.state.searchTerm}
            />
        </div>

      </RatingsAndReviewsContainer>
    </section>
    );
}
}






export default RatingsReviews;


