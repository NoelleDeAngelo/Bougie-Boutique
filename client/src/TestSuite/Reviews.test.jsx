import React from 'react'
import RatingsReviews from '../components/RatingsReviews/index.jsx'
import {render} from '@testing-library/react'
import {light} from '../Styles/themes.jsx'

var store ={};
store.setMeta = ()=>{}
store.setReviews = ()=>{}
store.state = {
  "currentProductId": "13023",
  "product": {
      "name": "Camo Onesie",
  },

  "meta": {
      "ratings": {
          "1": "3",
          "3": "4",
          "4": "5",
          "5": "11"
      },
      "recommended": {
          "false": "3",
          "true": "20"
      },
      "characteristics": {
          "Fit": {
              "id": 43617,
              "value": "3.9583333333333333"
          }
      },
      "averageRating": 3.9,
      "starRating": 4
  },
  "reviews": [
      {
          "rating": 5,
          "date": "2021-03-08T00:00:00.000Z",
          "summary": "Makes me smile",
          "body": "This product is great because it puts a little smile on my face!",
          "photos": [
              {
                  "id": 496602,
                  "url": "https://i.ibb.co/5xbC4Xk/happy-Monkey.jpg"
              }
          ],
          "recommended": true,
          "reviewerName": "SmilingSean",
          "responseToReview": null,
          "helpfulness": 42
      }
  ],

}
test('<RatingsReviews/> renders without crashing', ()=>{
  var div = document.createElement('div')
  render(
    <RatingsReviews store = {store} theme={light} />, div
  )
});

test('reviews section matches snapshot', ()=>{
  var component = render(
    <RatingsReviews store = {store} theme={light} />
  )

  expect(component.container).toMatchSnapshot();
})


test('<RatingsReviews/> renders without crashing', ()=>{
  const div = document.createElement('div')
  const {getByTestId} = render(
    <RatingsReviews store = {store} theme={light} />, div
  )
  const content = getByTestId('rairings')

});


