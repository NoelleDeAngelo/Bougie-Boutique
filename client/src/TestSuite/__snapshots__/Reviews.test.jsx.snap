// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`reviews section matches snapshot 1`] = `
<div>
  <section
    data-testid="content"
    tracking="Ratings and Reviews"
  >
    <h2
      id="ratingsreviews"
      style="margin-bottom: 50px; margin-top: 100px; margin-left: 3%;"
    >
      Ratings and Reviews
    </h2>
    <section
      class="RatingsandReviews__RatingsAndReviewsContainer-sc-17lej04-0 ijbsWd"
    >
      <div
        class="RatingsandReviews__RatingsContainer-sc-17lej04-1 goVBSm"
      >
        <div>
          <span
            style="font-size: 300%;"
          >
            3.9
          </span>
          <div
            class="Stars__StarsOuter-sc-7428jg-0 eNEVLX"
            style="font-size: 95%; vertical-align: top; margin-left: 10px;"
          >
            <div
              class="Stars__StarsInner-sc-7428jg-1 ipzXih"
            />
          </div>
          <p>
            87
            % of reviews recomend this product
          </p>
          <div />
           
          <span
            class="ratingBar"
          >
            <span>
              5 stars 
            </span>
            <meter
              max="100"
              min="0"
              value="47.82608695652174"
            />
             
            <span
              style="font-size: 80%;"
            >
              (
              11
              )
            </span>
          </span>
          <br />
          <div />
          <span
            class="ratingBar"
          >
            <span>
              4 stars 
            </span>
            <meter
              max="100"
              min="0"
              value="21.73913043478261"
            />
             
            <span
              style="font-size: 80%;"
            >
              (
              5
              )
            </span>
          </span>
          <br />
          <div />
          <span
            class="ratingBar"
          >
            <span>
              3 stars 
            </span>
            <meter
              max="100"
              min="0"
              value="17.391304347826086"
            />
             
            <span
              style="font-size: 80%;"
            >
              (
              4
              )
            </span>
          </span>
          <br />
          <div />
          <span
            class="ratingBar"
          >
            <span>
              2 stars 
            </span>
            <meter
              max="100"
              min="0"
              value="0"
            />
             
            <span
              style="font-size: 80%;"
            >
              (
              0
              )
            </span>
          </span>
          <br />
          <div />
          <span
            class="ratingBar"
          >
            <span>
              1 stars 
            </span>
            <meter
              max="100"
              min="0"
              value="13.043478260869565"
            />
             
            <span
              style="font-size: 80%;"
            >
              (
              3
              )
            </span>
            <br />
          </span>
          <br />
          <div>
            <div
              style="margin-bottom: 50px; width: 250px; border-radius: 4px;"
            >
              <div>
                Fit
              </div>
              <i
                class="lni lni-diamond"
                style="margin-left: 147.91666666666669px;"
              />
              <div
                style="background: rgb(238, 238, 238); width: 250px; height: 7px;"
              />
              <span
                style="font-size: 90%; margin-top: 5px;"
              >
                Runs tight
              </span>
              <span
                style="float: right; font-size: 90%; margin-top: 5px;"
              >
                Runs long
              </span>
            </div>
          </div>
        </div>
      </div>
      <div>
        <div
          style="min-width: 300px; height: 30px; width: 40%; border: 2px solid black; margin: 20px 20px 20px 0px;"
        >
           
          <i
            class="lni lni-32 lni-search-alt"
            style="font-size: 95%; margin: 3px;"
          />
          <input
            placeholder="Search Reviews"
            style="height: 27px; width: 90%; outline: none; color: black;"
            type="text"
          />
        </div>
        <div
          class="reviews"
        >
          <span>
            1
             reviews, sorted by 
          </span>
          <select
            class="dropDown"
          >
             
            <option>
              Relevant
            </option>
            <option>
              Helpful
            </option>
            <option>
              Newest
            </option>
          </select>
           
          <br />
          <section
            style="max-height: 780px; overflow-y: auto;"
          >
            <div
              class="RatingsandReviews__SingleReview-sc-17lej04-2 ddUuHr"
            >
              <div
                class="Stars__StarsOuter-sc-7428jg-0 eNEVLX"
              >
                <div
                  class="Stars__StarsInner-sc-7428jg-1 fVhJIw"
                />
              </div>
              <span
                style="float: right;"
              >
                SmilingSean
                ,   
                March 7, 2021
              </span>
              <h3>
                Makes me smile
              </h3>
              <p>
                This product is great because it puts a little smile on my face!
              </p>
              <div>
                <img
                  src="https://i.ibb.co/5xbC4Xk/happy-Monkey.jpg"
                  style="width: 65px;"
                />
              </div>
              <p
                style="font-size: 90%;"
              >
                 âœ“ I recomend this product
              </p>
              <div />
              <div
                style="font-size: 90%;"
              >
                <span>
                  Was this review helpful? 
                </span>
                <a
                  style="margin: 10px;"
                >
                  Yes(
                  42
                  )
                </a>
                <span
                  style="margin-left: 10px; margin-right: 15px;"
                >
                  |
                </span>
                <a>
                  Report
                  <i
                    class="lni lni-flag-alt"
                  />
                   
                </a>
              </div>
            </div>
          </section>
          <div />
          <button
            style="font-size: 105%; border-radius: 1px; padding: 15px; margin: 10px; font-family: Josefin Sans; font-weight: bold; color: black; border: 3px solid; border-color: #002a60;"
          >
            Add A Review
          </button>
        </div>
      </div>
    </section>
  </section>
</div>
`;
